python_ta/config/messages_config.toml

```diff
@@ -13,6 +13,7 @@ W0199 = "Calling assert on a tuple will only evaluate to False if it is empty. I
 
 ["pylint.checkers.base".ComparisonChecker]
 C0123 = "You should use 'isinstance(x, my_type)' instead of 'type(x) == my_type' to check the type of a value."
+W0143 = "This expression includes a comparison against a callable. It's likely you meant to call a function but omitted the parenthesis."
```

Comment:
Let's use just the term "function" instead of "callable" in the description, as the latter isn't commonly taught for beginners.

And at the end, please say "parenthesis, ()" instead of just "parenthesis", just to be a bit more concrete.